cloudquery/README.md

```diff
@@ -0,0 +1,177 @@
+
+CloudQuery supports [OpenTelemetry][5] traces, metrics and logs out of the box.
+There are multiple ways to configure OpenTelemetry with Datadog. We'll show only a subset of them here, and you can find more information [in the Datadog docs][6].
+
+- Using an OpenTelemetry collector
+
+To configure an OpenTelemetry collector with Datadog, you need to create a configuration file, for example `otel_collector_config.yaml` with the content below:
+
+```yaml
+receivers:
+  otlp:
+    protocols:
+      http:
+        endpoint: "0.0.0.0:4318"
+
+processors:
+  batch/datadog:
+    send_batch_max_size: 1000
+    send_batch_size: 100
+    timeout: 10s
+
+exporters:
+  datadog:
+    api:
+      site: ${env:DATADOG_SITE}
+      key: ${env:DATADOG_API_KEY}
+
+service:
+  pipelines:
+    metrics:
+      receivers: [otlp]
+      processors: [batch/datadog]
+      exporters: [datadog]
+    traces:
+      receivers: [otlp]
+      processors: [batch/datadog]
+      exporters: [datadog]
+    logs:
+      receivers: [otlp]
+      processors: [batch/datadog]
+      exporters: [datadog]
+```
+
+Then run the collector with the following command (replacing `DATADOG_SITE` and `DATADOG_API_KEY` with your own values):
+
+```bash
+docker run \
+    -p 4318:4318 \
+    -e DATADOG_SITE=$DATADOG_SITE \
+    -e DATADOG_API_KEY=$DATADOG_API_KEY \
+    --hostname $(hostname) \
+    -v $(pwd)/otel_collector_config.yaml:/etc/otelcol-contrib/config.yaml \
+    otel/opentelemetry-collector-contrib:0.104.0
+```
+
+> For additional ways to run the collector, please refer to the [official documentation][7].
+
+- Direct OTEL Ingestion by the Datadog Agent via a configuration file
+
+[Locate][8] your `datadog.yaml` file and add the following configuration:
+
+```yaml
+otlp_config:
+  receiver:
+    protocols:
+      http:
+        endpoint: 0.0.0.0:4318
+  logs:
+    enabled: true
+logs_enabled: true
+```
+
+[Restart][9] the Datadog agent for the change to take effect.
+
+- Direct OTEL ingestion by the Datadog Agent via environment variables
+
+Pass the `DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_HTTP_ENDPOINT` environment variable to the Datadog agent with a value of `0.0.0.0:4318`.
+If you're using Docker compose, you can find an example below:
+
+```yaml
+version: "3.0"
+services:
+  agent:
+    image: gcr.io/datadoghq/agent:7
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock:ro
+      - /proc/:/host/proc/:ro
+      - /sys/fs/cgroup/:/host/sys/fs/cgroup:ro
+    environment:
+      DD_API_KEY: redacted
+      DD_SITE: "datadoghq.eu"
+      DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_HTTP_ENDPOINT: "0.0.0.0:4318"
+      DD_LOGS_ENABLED: "true"
+      DD_OTLP_CONFIG_LOGS_ENABLED: "true"
+    ports:
+      - "4318:4318"
+```
+
+[Restart][10] the Datadog agent for the change to take effect.
+
+> For additional ways to configure the Datadog agent, please refer to the [official documentation][11].
+
+Once you have the agent or collector ready, you can specify the endpoint in the source spec:
```

Comment:
Is this optional? If not, recommend moving this to each configuration as the final step. For example: 
```
1. Restart the Datadog agent for the change to take effect.
2. After you have the Agent ready, specify the endpoint in the source spec:
```
```suggestion
After you have the Agent or collector ready, specify the endpoint in the source spec:
```

Reply:
This is required, good suggestion 👍